client/src/components/App.js

```diff
@@ -207,6 +229,30 @@ export default class App extends React.Component {
       body: JSON.stringify(data)
     });
 
+  typingMessageLang = lang => {
+    if (lang === 'en') {
```

Comment:
is it worth changing to objects with key value pairs now? If not then these could be one-liners
```js
if (lang === 'en') return 'typing...';
else if (lang === 'fr) return 'en train de taper...';
```

Reply:
same below :point_down: